docs/docs/Integrations/Composio/integrations-composio.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Integrate Composio with Langflow
+slug: /integrations-composio
+---
+
+Langflow integrates with [Composio](https://docs.composio.dev/introduction/intro/overview) as a toolset for your **Agent** component.
+
+Instead of juggling multiple integrations and components in your flow, connect the Composio component to an **Agent** component to use all of Composio's supported APIs and actions as **Tools** for your agent.
+
+## Prerequisites
+
+- [A Composio API key created](https://app.composio.dev/)
+- [An OpenAI API key created](https://platform.openai.com/)
+- [A Gmail account created](mail.google.com)
+
+## Connect Langflow to a Composio tool
+
+1. In the Langflow **Workspace**, add an **Agent** component.
+2. In the **Workspace**, add the **Composio Tools** component.
+3. Connect the **Agent** component's **Tools** port to the **Composio Tools** component's **Tools** port.
+4. In the **Composio API Key** field, paste your Composio API key.
+Alternatively, add the key as a [global variable](/configuration-global-variables).
+5. In the **App Name** field, select the tool you want your agent to have access to.
+For this example, select the **GMAIL** tool, which allows your agent to control an email account with the Composio tool.
+6. Click **Refresh**.
+The component's fields change based on the tool you selected.
+The **Gmail** tool requires authentication with Google, so it presents an **Authentication Link** button.
+7. Click the link to authenticate.
+A new window opens.
```

Comment:
Don't think we need to say this. However, you could add text to the beginning of step 8 to say "In the **name of the new dialog**, enter the credentials..."